e2e/bzlmod/.bazelrc

```diff
@@ -2,4 +2,3 @@
 import %workspace%/../../.bazelrc.common
 
 common --experimental_enable_bzlmod
-common --registry=https://raw.githubusercontent.com/aspect-build/bazel-central-registry/main/
```

Comment:
BCR is pretty responsive with merges not, and they migrated it to be compatible with `5.3.0` so I don't think we need to use our own registry.